geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/ClientRegistrationEventQueueManager.java

```diff
@@ -37,15 +39,25 @@
  */
 public class ClientRegistrationEventQueueManager {
   private static final Logger logger = LogService.getLogger();
+
   private final Set<ClientRegistrationEventQueue> registeringProxyEventQueues =
       ConcurrentHashMap.newKeySet();
 
   void add(final InternalCacheEvent event,
+      final ClientUpdateMessageImpl clientMessage,
       final Conflatable conflatable,
       final Set<ClientProxyMembershipID> originalFilterClientIDs,
       final CacheClientNotifier cacheClientNotifier) {
-    if (registeringProxyEventQueues.isEmpty())
+    if (registeringProxyEventQueues.isEmpty()) {
       return;
+    }
+
+    if (originalFilterClientIDs.isEmpty()) {
```

Comment:
How about combine all the conditions into a single `if` statement, instead of two?

Reply:
Fixed!